crates/circuit/src/variable_mapper.rs

```diff
@@ -43,20 +43,20 @@ impl<'py> FromPyObject<'py> for Condition {
 ///
 /// TODO: move this to control flow mod once that's in Rust.
 #[derive(IntoPyObject)]
-pub(crate) enum Target {
+pub(crate) enum ControlFlowTarget {
```

Comment:
I think this is specifically the `SwitchTarget` - can we call it that instead?  I was confused at first, because it can't be the "target" of an `if` statement (the `Bit` variant would need to store the True/False state - turns out that part is called `Condition`).

Reply:
Sure, I'll change it to `SwitchTarget` I wasn't 100% sure so I left it more generic

Reply:
Done in: https://github.com/Qiskit/qiskit/pull/14935/commits/a55462e8a23f198c6cb17ae78ce2e5d9f3b7c9f9